codespell_lib/data/dictionary.txt

```diff
@@ -25141,7 +25149,7 @@ standartized->standardized
 standarts->standards
 standatd->standard
 standtard->standard
-standy->standby
+standy->standby, sandy,
```

Comment:
Maybe standee?

Reply:
I thought I pondered that before, not sure why it didn't make it in.